platform/stm32f4xx/export/aux/dma_device.hpp

```diff
@@ -104,16 +144,13 @@ constexpr auto get_ht_flag()
         return DMA_FLAG_HTIF6;
     case 7:
         return DMA_FLAG_HTIF7;
-    default:
-        return 0xfful; // TODO
     }
```

Comment:
Reason is that when it comes to constant expressions compiler will emit error if it cannot compute (or evaluate at compile time) certain  parts of the expression. It is way to produce [implicit static assertions](https://godbolt.org/g/kfTVQq)

Hopefully, one day [explicit static asserts](http://en.cppreference.com/w/cpp/language/static_assert) will have some similar mechanism so it will be possible to put them at `default` case